functions/icons.zsh

```diff
@@ -86,6 +87,76 @@ case $POWERLEVEL9K_MODE in
   'awesome-fontconfig')
     # fontconfig with awesome-font required! See
     # https://github.com/gabrielelana/awesome-terminal-fonts
+    # if not defined, set recommended linux path
+    typeset -p "POWERLEVEL9K_FONTAWESOME_PATH" > /dev/null 2>&1  || POWERLEVEL9K_FONTAWESOME_PATH=~/.fonts
```

Comment:
@gabrielelana - Okay, awesome. Can you let us know once the Awesome docs are updated? If I merge this before that's done, we are both likely to get hammered with support issues :)